fix compile Probs up from vdr-1.5.3

Joerg Bornkessel <hd_brummy@gentoo.org> 2007 06 26

diff -Naur graphlcd-0.1.5.orig/layout.c graphlcd-0.1.5/layout.c
--- graphlcd-0.1.5.orig/layout.c	2007-06-26 21:43:52.000000000 +0200
+++ graphlcd-0.1.5/layout.c	2007-06-26 21:46:05.000000000 +0200
@@ -65,7 +65,11 @@
             file += "/fonts/";
             file += url.substr(4, pos - 4);
         }
+#if APIVERSNUM >= 10503
+		return font.LoadFT2(file, cCharSetConv::SystemCharacterTable(), size);
+#else
         return font.LoadFT2(file, I18nCharSets()[Setup.OSDLanguage], size);
+#endif
     }
     else
     {
